SERVER/main.py

```
@@ -1,18 +1,75 @@
+    print("Starting Gradio and FastAPI servers in separate processes...")
+
+    gradio_process = multiprocessing.Process(target=run_gradio, name="GradioInterface")
+    fastapi_process = multiprocessing.Process(target=run_fastapi, name="FastAPIServer")
+
+    gradio_process.start()
+    fastapi_process.start()
+
+    print(f"Gradio process started with PID: {gradio_process.pid}")
+    print(f"FastAPI process started with PID: {fastapi_process.pid}")
+
+    def shutdown_handler(signum, frame):
```

Ruff (0.11.9)

Missing return type annotation for private function `shutdown_handler`

Add return type annotation: `None`

(ANN202)